lib/kb_Metrics/metricsdb_controller.py

```diff
@@ -30,9 +30,46 @@ def _config_str_to_list(self, list_str):
 
         return user_list
 
+    def _parse_app_id(self, exec_task):
+        app_id = ''
+        if 'app_id' in exec_task['job_input']:
+            app_id = exec_task['job_input']['app_id']
+            app_id = app_id.replace('.', '/')
+
+        return app_id
+
+    def _parse_method(self, exec_task):
+        method_id = ''
+        if 'method' in exec_task['job_input']:
+            method_id = exec_task['job_input']['method']
+            method_id = method_id.replace('/', '.')
```

Comment:
same here

Reply:
'method' may not always exist in exec_task['job_input'].

Reply:
same here. You don't need to have method_id twice.